framework/source/class/qx/test/log/Filters.js

```diff
@@ -0,0 +1,64 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2004-2009 1&1 Internet AG, Germany, http://www.1und1.de
+
+   License:
+     LGPL: http://www.gnu.org/licenses/lgpl.html
+     EPL: http://www.eclipse.org/org/documents/epl-v10.php
+     See the LICENSE file in the project's top-level directory for details.
+
+   Authors:
+ * Fabian Jakobs (fjakobs)
+
```

Comment:
Wouldn't it make sense to start using the newly suggested MIT license?

See http://news.qooxdoo.org/license-update

On Thu, 18 Feb 2016, Tobias Oetiker wrote:

>> +
>> +   qooxdoo - the new era of web development
>> +
>> +   http://qooxdoo.org
>> +
>> +   Copyright:
>> +     2004-2009 1&1 Internet AG, Germany, http://www.1und1.de
>> +
>> +   License:
>> +     LGPL: http://www.gnu.org/licenses/lgpl.html
>> +     EPL: http://www.eclipse.org/org/documents/epl-v10.php
>> +     See the LICENSE file in the project's top-level directory for details.
>> +
>> +   Authors:
>> + * Fabian Jakobs (fjakobs)
>> +
>
> author would be you, right ?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/qooxdoo/qooxdoo/pull/152/files#r53308158

-- 
Oetiker+Partner AG              tel: +41 62 775 9903 (direct)
Fritz Zaucker                        +41 62 775 9900 (switch board)
Aarweg 15                            +41 79 675 0630 (mobile)
CH-4600 Olten                   fax: +41 62 775 9905
Schweiz                         web: www.oetiker.ch